.changeset/many-boats-peel.md

```diff
@@ -0,0 +1,8 @@
+---
+'create-astro': minor
+---
+
+Alway write chosen config to `tsconfig.json`.
+
+- Before: Only when `strict` & `strictest` was selected
+- After: Also when `base` is selected (via "Relaxed" or "I prefer not to use TypeScript")
```

Comment:
I wonder whether users would be confused as to why they have an extra file in their project when they chose "no typescript"? I suppose they can just delete it later..

Reply:
> I wonder whether users would be confused as to why they have an extra file in their project when they chose "no typescript"? I suppose they can just delete it later..

We're planning to update the message soon so it's more clear!